tests/book.test.js

```diff
@@ -0,0 +1,141 @@
+const { expect } = require("chai");
+const request = require("supertest");
+const { Book } = require("../src/models");
+const app = require("../src/app");
+
+describe("/books", () => {
+  before(async () => Book.sequelize.sync());
+
+  beforeEach(async () => {
+    await Book.destroy({ where: {} });
+  });
+  describe("with no books on database", () => {
+    describe("POST /books", () => {
+      it("creates a new book in the database", async () => {
+        const response = await request(app).post("/books").send({
+          title: "Redwall",
+          author: "Brian Jacques",
+          ISBN: "72A",
+        });
+
```

Comment:
remove clear line and add it after line 23 to group execution and assertions together like you have done in the tests at the end of this file - consistency